app/controllers/application_controller.rb

```diff
@@ -6,7 +6,7 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
```

Comment:
At the very top of this file add this code and it will fix the code climate error.  
 # frozen_string_literal: true

Reply:
Thanks, made the update.